RFC-010-core-strategy/core-strategy.md

```diff
@@ -0,0 +1,67 @@
+# Executive Summary
+
+The Good Docs Project has grown organically since its creation and has blossomed into a thriving community. Now it’s important to articulate where we are, how we can best support our users and community members, and where we’d like to grow in the future. 
```

Comment:
Which year was it created in? List that here?

Reply:
Added document history section.